File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        ans = [0] * n
        num = n//2
        idx = 0 
        # while num > 0:
        #     ans[idx] = num
        #     ans[n-idx-1] = -num
        #     num -= 1
        #     idx += 1  
        
        for i in range(-(n//2), n//2+1, 1):
            if i==0 and n%2==0:
                continue;   
            ans[idx] = i
            idx+=1
            
        return ans
        

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.java
class Solution {
    public int[] sumZero(int n) 
	{
         int[] a = new int[n];
         int p = -(n / 2);
         if(n % 2 == 1)
		 {
        	 for(int i=0; i<a.length; i++)
			 {
        		 a[i] = p;
        		 p += 1;
        	 }
         }
         else
		 {
        	 for(int i=0; i<a.length; i++) 
			 {
        		 a[i] = p;
        		 p += 1;
        		 if(p == 0)
				 {
        			 p ++;
        		 }
        	 }
         }
         return a;
    }
}